frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -42,16 +42,16 @@ const AlertRouting = () => {
       <div className="row">
         <div className="col-sm-6">
           <dt>Group By</dt>
-          <dd>{_.isEmpty(groupBy) ? '-' : _.join(groupBy, ', ')}</dd>
+          <dd id="group_by_value">{_.isEmpty(groupBy) ? '-' : _.join(groupBy, ', ')}</dd>
```

Comment:
Use `data-test-id` instead of `id` (here and below)

Reply:
fixed